src/utils/toErrorString.ts

```diff
@@ -0,0 +1,11 @@
+const toErrorString = (error: unknown) => {
+  if (error instanceof Error) {
+    return error.message;
+  }
+  if (typeof error === "string") {
+    return error;
+  }
+  return "Unknown error";
```

Comment:
Was wondering if it might be a slight improvement if we make an `enum` out of this? This way, if we change this error naming in the future, we won't need to manually change the error naming in all the respective test files where it is used, as we will just use the exported `enum` instead.

Reply:
Yes, good idea. Done!